fs-repo-14-to-15/migration/migration.go

```diff
@@ -172,65 +172,79 @@ func convert(in io.Reader, out io.Writer) error {
 	}
 
 	// Remove /quic only addresses from .Addresses.Swarm
-	if a, ok := confMap["Addresses"]; ok {
-		addresses, ok := a.(map[string]any)
-		if !ok {
-			return fmt.Errorf("invalid type for .Addresses got %T expected json map", a)
-		}
-
-		for _, addressToRemove := range [...]string{"Swarm", "Announce", "AppendAnnounce", "NoAnnounce"} {
-			s, ok := addresses[addressToRemove]
+	if err := func() error {
+		if a, ok := confMap["Addresses"]; ok {
+			addresses, ok := a.(map[string]any)
 			if !ok {
-				continue
+				fmt.Printf("invalid type for .Addresses got %T expected json map; skipping .Addresses\n", a)
+				return nil
 			}
```

Comment:
What kinds of things are we expecting here? Is the idea that we don't really care, it's not something with a valid `/quic` address we can deal with?

Reply:
mb should be a continue